cvat-core/src/annotations-objects.js

```diff
@@ -1383,6 +1472,7 @@
             super(data, clientID, color, injection);
             this.shapeType = ObjectShape.RECTANGLE;
             this.pinned = false;
+            this.annotation_type = AnnotationType.MANUAL;
```

Comment:
Why tracks cannot be auto annotated?

Reply:
As stated before, this was implemented for TF auto annotation API. Also, the value taken from data is undefined in some cases.